rcublas/cublas/src/error.rs

```diff
@@ -32,14 +32,14 @@ impl fmt::Display for Error {
         let msg = match *self {
```

Comment:
in 2018 we should not need the `*` and `refs` anymore, do we?

Reply:
Haven't tracked it, but it sounds likely from NLL. We can open a chore issue later and resolve it at some point